workloads/ric/README.md

```diff
@@ -0,0 +1,52 @@
+# RIC packages
+
+Packages for deploying the following are stored in the folder
+
+- RIC custom controller
+- RIC
+- Workload cluster for RIC deployments
+
+PackageVariant and PackageVariantSets for deploying RIC are also stored in this folder.
+
+# Steps to manually deploy RIC on Nephio
+
+## Step 0: Prerequisite
+
+1. Nephio should be installed.
+2. Since RIc will be deployed on Regional Cluster regional cluster should be running.

```

Comment:
@naman-gupta-641 I think it will be better to provide a link, the free5gc exercise version does not have regional cluster. 

1. Install Nephio as per the [user guide](https://github.com/nephio-project/docs/blob/main/content/en/docs/guides/install-guides/_index.md)
2. Follow the [exercise to install OAI components](https://github.com/nephio-project/docs/blob/main/content/en/docs/guides/user-guides/exercise-2-oai.md). RIC will be installed on the regional cluster. 
3. Register the RIC repository `kpt alpha repo register --namespace default https://github.com/nephio-project/catalog.git --directory=workloads/ric`

Reply:
done